### PR TITLE
tests for AppContentSwitcher component

### LINKED ISSUE #99: (SP: 1) Write unit test for "AppContentSwitcher" component
### Component unit test
Unit test for **"AppContentSwitcher"** component
Scenaries descriptions:
 - [ ] Should render correctly with props
 - [ ] Should call the function "onChange" when it was clicked on the switch
 - [ ] Should render tooltips if the tooltips props are passed

[Link to component](https://github.com/Space2Study-UA-1156/Client-01/blob/develop/src/components/app-content-switcher/AppContentSwitcher.jsx)

Current coverage:
image

### PR DESCRIPTION
**Pull request by OlhaOvsiienko**
Here are tests for AppContentSwitcher component.
image
Thanks!

### CODE REVIEW

**Review by khytrykv — APPROVED** [review]

**Review by holus-bolus — APPROVED** [review]

**Review by YaroslavChuiko — APPROVED** [review]

File: tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        onChange={onChangeMock}
+        styles={{}}
+        switchOptions={{}}
+        typographyVariant='body1'
+      />
+    )
+
+    const switchElement = screen.getByTestId('switch')
+    fireEvent.click(switchElement)
+  })
```
Comment (vlad-khrychov) [inline tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx]: Add `expect` to this test

File: tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import { render, screen, fireEvent } from '@testing-library/react'
+import { describe, it, beforeEach } from 'vitest'
```
Comment (vlad-khrychov) [inline tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx]:
```suggestion
```

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

File: tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx (hunk 2 above)
Comment (OlhaOvsiienko) [reply tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx]: Unnecessary import was deleted.

Comment (OlhaOvsiienko) [reply tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx]: The 'expect' was added in the 2nd test, which checks if the function has been called.

File: tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx (hunk 1 above)
Comment (OlhaOvsiienko) [reply tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx]: The 'expect' was added in the 2nd test, which checks if the function has been called.

**Review by YaroslavChuiko — APPROVED** [review]

**Review by voloshynvv — APPROVED** [review]

**Review by khytrykv — APPROVED** [review]

File: tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+describe('AppContentSwitcher tests', () => {
+  let switchOptions
+
+  beforeEach(() => {
+    switchOptions = {
+      left: { text: 'Left Text', tooltip: 'Left Tooltip' },
+      right: { text: 'Right Text', tooltip: 'Right Tooltip' }
+    }
+  })
```
Comment (vlad-khrychov) [inline tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx]:
```suggestion
  const switchOptions = {
      left: { text: 'Left Text', tooltip: 'Left Tooltip' },
      right: { text: 'Right Text', tooltip: 'Right Tooltip' }
    }
```

**Review by vlad-khrychov — APPROVED** [review]
We may leave `beforeEach`, but I think `const` could be enough.

**Pull request merged**